cluster-provision/k8s/1.17/provision.sh

```diff
@@ -2,6 +2,14 @@
 
 set -ex
 
+# Resize root partition
+dnf install -y cloud-utils-growpart
+growpart /dev/vda 1
+xfs_growfs -d /
+
+# Set hostname right
+dhclient
```

Comment:
We might need this in every reboot this vm will do

Reply:
I forgot to remove this, we are using dhclient as dhcp option in NM.